### PR TITLE
DDCE-1978: govuk Assets upgrade and added missing welsh

### PR DESCRIPTION
**Pull request by fraser-sprott**

### CODE REVIEW

File: test/unit/views/behaviours/ViewBehaviours.scala [hunk 1]
```diff
@@ -49,13 +48,12 @@ trait ViewBehaviours extends ViewSpecBase {
           val doc = asDocument(view())
           for (key <- expectedGuidanceKeys) assertContainsText(doc, messages(s"$messageKeyPrefix.$key"))
         }
-
         "contain a timeout dialog" in {
-          val doc = asDocument(view())
+          val timeoutElm = asDocument(view()).select("meta[name=\"hmrc-timeout-dialog\"]")
           if(expectTimeoutDialog) {
-            assertRenderedById(doc, "timeoutDialog")
+            timeoutElm.size == 1
```
Comment (JMaio) [inline test/unit/views/behaviours/ViewBehaviours.scala:54]: is this check asserted?

Comment (JMaio) [reply test/unit/views/behaviours/ViewBehaviours.scala:54]: does `==` behave in the same way as `shouldBe` or another assertion? it looks like it'll just return `true` or `false` as opposed to failing the test

Comment (fraser-sprott) [reply test/unit/views/behaviours/ViewBehaviours.scala:54]: nope, cleary not fully awake yet, will fix it

**Review by JMaio — APPROVED** [review]
preliminary approval - would benefit from a second reviewer

**Review by yangwong9664 — CHANGES_REQUESTED** [review]
There are a couple of warnings:

connectors/EmailConnector.scala:37:80: parameter value rds in method send is never used
[warn]   def send[E >: Email[_]](e: E)(implicit hc: HeaderCarrier, writes: Writes[E], rds: Reads[E]): 
                                                                          
utils/JsonFormatters.scala:50:68: pattern var obj in value $anonfun is never used; `obj@_' suppresses this warning

                                                                 ^
connectors/EmailConnector.scala:40:18: value readRaw in object HttpReads is deprecated (since 11.0.0): Use uk.gov.hmrc.http.HttpReads.Implicits instead. See README for differences.
[warn]       client.POST(url = url, body = e).map(_ => ())

There's also a few Scalastyle warnings

**Review by yangwong9664 — APPROVED** [review]

**Review by JMaio — APPROVED** [review]

**Pull request merged**